packages/office-ui-fabric-react/src/components/Facepile/Facepile.types.ts

```diff
@@ -122,6 +123,11 @@ export interface IFacepilePersona extends React.ButtonHTMLAttributes<HTMLButtonE
    * handlers.
    */
   data?: any;
+
+  /**
+   * Optional keytip for this button
```

Comment:
I would say here in the comment that the keytip will only be added if the onClick is defined (because I think that's the only way it gets into creating a FacepileButton correct?)

Reply:
Yes that is correct, I'll add this now.